lib/utilities/pristine.js

```diff
@@ -85,7 +86,9 @@ function installPristineApp(appName, options) {
 
   chdir(temp.pristinePath);
 
-  var promise = applyCommand('new', appName, extraOptions)
+  var args = extraOptions.concat(['-sb', '-sn', runCommandOptions]);
```

Comment:
Seems you are right, indeed. But then we can get rid of the `extraOptions`, as they don't make any difference, right?